### DISCUSSION TITLE
Upgrade from jnv/ansible-role-unattended-upgrades - stray 50unattended-upgrades?

### DISCUSSION THREAD

**Discussion by dnmvisser** [hifis-net/ansible-collection-toolkit#191] [Q&A] [answered]
Hi, I am about to migrate from https://github.com/jnv/ansible-role-unattended-upgrade to this role. Just replacing the role seems to cause a stray `50unattended-upgrades` file to be left in `/etc/apt/apt.conf.d` (the new file is called `90-ansible-unattended-upgrades`).
I can manually remove `50unattended-upgrades`, but I didn't find this in the docs. 
Can someone confirm that this is indeed something that has to be done?

**Comment by tobiashuste** [comment, marked as answer]
The behavior was changed in version `2.0.0` (https://github.com/hifis-net/ansible-role-unattended-upgrades/releases/tag/v2.0.0) of the role in which the system provided variants of the configuration were no longer updated. Instead, all configuration was placed in a file which is entirely controlled via this role. In version `3.0.0` (https://github.com/hifis-net/ansible-role-unattended-upgrades/releases/tag/v3.0.0) some additional changes were implemented. The release notes document the behavioral changes.

There is no need for a user to touch any of the system provided configuration. Files can be left untouched. Since no action is required by the user, this was not added to the documentation so far.
